cmd/wallet-web/src/store/modules/user.js

```diff
@@ -217,12 +222,16 @@ export default {
       return state.selectedCredentialId;
     },
     getGnapSessionToken(state) {
-      state.gnapSessionToken = localStorage.getItem('gnapSessionToken');
-      return JSON.parse(state.gnapSessionToken);
+      state.gnapSessionToken = JSON.parse(localStorage.getItem('gnapSessionToken'));
+      return state.gnapSessionToken;
+    },
+    getGnapSubjectId(state) {
+      state.gnapSubjectId = JSON.parse(localStorage.getItem('gnapSubjectId'));
+      return state.gnapSubjectId;
     },
     getGnapReqAccessResp(state) {
-      state.gnapRequestAccessrResp = localStorage.getItem('gnapRequestAccessrResp');
-      return JSON.parse(state.gnapRequestAccessrResp);
+      state.gnapRequestAccessResp = JSON.parse(localStorage.getItem('gnapRequestAccessResp'));
```

Comment:
why you read this from local storage everytime inside getters?
also, vuex getters shouldn't update state.

Typically, getters shouldn't hit local storage. Imagine hitting localstorage everytime this getter is called. I see in `router/index.js` you are calling this getter during each navigation.